prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -398,10 +406,18 @@ periodics:
     - <<: *kyma_ref
       base_ref: master
   spec:
+    volumes:
+      - name: sa-stability-fluentd-storage-writer
+        secret:
+          secretName: sa-stability-fluentd-storage-writer
```

Comment:
this secret should be added to https://github.com/kyma-project/test-infra/blob/master/prow/cluster/required-secrets.yaml. After merging to the master let me know about it, I will propagate storage

Reply:
👍